src/main/java/frc/robot/Robot.java

```diff
@@ -277,6 +266,11 @@ public void configureButtonBindings() {
 
     @Override
     public void populateDashboard() {
+        Shuffleboard.getTab("Pit Test");
+        Shuffleboard.getTab("Pit Test").add("Drive Backward", drive.moveForDirectional(0, 1, 5));
+        Shuffleboard.getTab("Pit Test").add("Drive Left", drive.moveForDirectional(-1, 0, 5));
+        Shuffleboard.getTab("Pit Test").add("Drive Right", drive.moveForDirectional(1, 0, 5));
+        Shuffleboard.getTab("Pit Test").add("Drive Forward", drive.moveForDirectional(0, -1, 5));
```

Comment:
Awesome,now can you specify a layout for these so they make a plus shape? Maybe make the center a stop button?